Userland/Libraries/LibWeb/UIEvents/CompositionEvent.idl

```diff
@@ -0,0 +1,17 @@
+#import <UIEvents/UIEvent.idl>
+
+// https://w3c.github.io/uievents/#compositionevent
+[Exposed=Window]
+interface CompositionEvent : UIEvent {
+    constructor(DOMString type, optional CompositionEventInit eventInitDict = {});
+    readonly attribute USVString data;
+
+    // https://w3c.github.io/uievents/#dom-compositionevent-initcompositionevent
+    // FIXME: The spec uses WindowProxy rather than Window (spec bug?).
```

Comment:
I've opened a PR on the spec to see if this can be changed. https://github.com/w3c/uievents/pull/379

I did some research and Firefox and Chromium use Window (Safari uses WindowProxy for all UIEvents I checked, even where the spec specifies Window).

Reply:
Could you link the spec issue in the code? It will help organize tracking of upstream spec links in the near future

Reply:
Good idea - I'll do that :)